my calc/my calculator.c

```diff
@@ -0,0 +1,68 @@
+#include <stdio.h>
```

Comment:
Put the files (image and code) in the appropriate directory (C folder) and edit that folder readme with your entry (you will understand once you look at it)